azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/features/DeploymentApiLiveTest.java

```diff
@@ -0,0 +1,129 @@
+      parameters = "{\"newStorageAccountName\":{\"value\":\"" + resourceName + "\"},\"storageAccountType\":{\"value\":\"Standard_LRS\"},\"location\":{\"value\":\"West US\"}}";
+      properties = getPutBody(template, "Incremental", parameters);
+   }
+
+   private String getPutBody(String template, String mode, String parameters) {
+      String body = "{ " +
+              "\"properties\" : " +
+              "  { " +
+              "    \"template\" : " + template + ", " +
+              "    \"mode\" : \"" + mode + "\", " +
+              "    \"parameters\" : " + parameters + " " +
+              "  } " +
+              "}";
+      return body;
+   }
+
+   @Test(groups = "live")
+   public void testCreate() {
+      Deployment deployment = api().create(deploymentName, properties);
+      assertNotNull(deployment);
+
+      //Poll until resource is ready to be used
+      boolean jobDone = Predicates2.retry(new Predicate<String>() {
+         @Override
+         public boolean apply(String name) {
+            Deployment dp = api().get(deploymentName);
+            ProvisioningState state = ProvisioningState.fromValue(dp.properties().provisioningState());
+            return state == ProvisioningState.SUCCEEDED;
+         }
+      }, 60 * maxTestDuration * 1000).apply(deploymentName);
+      assertTrue(jobDone, "create operation did not complete in the configured timeout");
+
+      Deployment dp = api().get(deploymentName);
+      ProvisioningState state = ProvisioningState.fromValue(dp.properties().provisioningState());
+      assertTrue(state == ProvisioningState.SUCCEEDED);
+   }
+
+
+   @Test(groups = "live", dependsOnMethods = "testCreate")
+   public void testGetDeployment() {
+      Deployment deployment = api().get(deploymentName);
+      assertNotNull(deployment);
+      ProvisioningState state = ProvisioningState.fromValue(deployment.properties().provisioningState());
+      assertTrue(state == ProvisioningState.SUCCEEDED);
+   }
+
+   @Test(groups = "live", dependsOnMethods = "testCreate")
+   public void testListDeployments() {
+      List<Deployment> deployments = api().list();
+      assertTrue(deployments.size() > 0 && deployments.get(0).name().equals(deploymentName));
```

Comment:
If the live tests are run in an account that already has data, is it correct to expect the one used in the tests to be returned in the first position?